android/src/main/java/io/radar/capacitor/RadarPlugin.java

```diff
@@ -476,9 +476,7 @@ public void isTracking(PluginCall call) {
     @PluginMethod()
     public void getTrackingOptions(PluginCall call) {
         RadarTrackingOptions options = Radar.getTrackingOptions();
-        JSObject ret = new JSObject();
-        ret.put("options", options != null ? options.toJson() : null);
-        call.resolve(ret);
+        call.resolve(RadarPlugin.jsObjectForJSONObject(options.toJson()));
```

Comment:
@lmeier Can you double check that this new return style is:
1) Consistent with the rest of the capacitor sdk (do we need fix this anywhere else?)
2) Consistent with how react native and flutter implement `getTrackingOptions()` and `getTripActions()`

Reply:
@lmeier Do we need to fix https://github.com/radarlabs/capacitor-radar/blob/master/src/definitions.ts ?

Or this bug fix now makes those definitions correct?

Reply:
Verified that the rest of the SDK is consistent, and that type definitions are now correct

Reply:
Confirmed that react native is the same (correct).

Reply:
Flutter is also correct.